src/styles/blocks/header.scss

```diff
@@ -0,0 +1,22 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  box-sizing: border-box;
+  padding: 0 50px;
+  margin: auto;
```

Comment:
According to the [Code Style Rules ❗️](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), `margin: auto;` is usually used to center block elements horizontally. However, in the context of a flex container, this property is not necessary for centering, as you've already set `justify-content: space-between;`. If the intention was to center the header itself, consider applying the `margin: auto;` to a wrapper element or using other centering techniques that are more explicit.